tests/jballerina-unit-test/src/test/resources/testng.xml

```diff
@@ -21,6 +21,13 @@
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
 <suite name="ballerina-new-parser-test-suite" time-out="120000">
+    <test name="jballerina-test-parallel" parallel="true">
```

Comment:
Need to revert this

Reply:
Done by https://github.com/ballerina-platform/ballerina-lang/pull/34900/commits/3f999ff9a99dd84536bb9359e7f1bd4ec3abe011